client/src/featureform/register.py

```diff
@@ -234,6 +331,33 @@ def register_resources(
             labels: List[ColumnMapping] = None,
             timestamp_column: str = ""
     ):
+        """
+        Registers a features and/or labels that can be used in training sets or served.
+
+        **Examples**:
+        ``` py
+        average_user_transaction.register_resources(
+            entity=user,
+            entity_column="CustomerID",
+            inference_store=local,
+            features=[
+                {"name": "avg_transactions", "variant": "quickstart", "column": "TransactionAmount", "type": "float32"},
+            ],
+        )
+        ```
+
+        Args:
+            entity (Union[str, EntityRegistrar]): The name to reference the entity by when serving features
+            entity_column (str): The name of the column in the source to be used as the entity
+            owner (Union[str, UserRegistrar]): The owner of the resource(s)
+            inference_store (Union[str, OnlineProvider]): Where to store the materialized feature for serving. (Use the local provider in Localmode)
+            features (List[ColumnMapping]): A list of column mappings to define the features
```

Comment:
is it obvious what column mapping means generally?

Reply:
Added a comment to the example above